Library/Homebrew/cmd/install.rb

```diff
@@ -64,6 +64,9 @@ def install_args
         description: "Don't delete the temporary files created during installation."
       switch "--build-bottle",
         description: "Prepare the formula for eventual bottling during installation."
+      switch "--bottle-arch=",
```

Comment:
I know I've wandered into new territory here (we're not in homebrew/homebrew-core anymore, Toto!) but I have a question:

Should there be an `=` here? Does this imply that `--bottle-arch` takes an argument? If so, what are valid values to pass here?

Reply:
I think the values in the following list are valid values:

https://github.com/Homebrew/brew/blob/master/Library/Homebrew/test/hardware/cpu_spec.rb#L18-L33

Reply:
Thanks for enlightening my ignorance!

Reply:
Yes, the CLI parser itself doesn't handle a list of valid arguments but that would be a good PR 👍